### PR TITLE
IGNITE-19575 Extend jmh benchmarks to compare thin clients against its embedded counterparts

### PR DESCRIPTION
**Pull request by korlov42**

### CODE REVIEW

File: modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/InsertBenchmark.java [hunk 1]
```diff
@@ -136,7 +159,54 @@ void executeQuery() {
...
+
+        /**
+         * Initializes session and statement.
+         */
+        @Setup
+        public void setUp() {
+            String fieldsQ = IntStream.range(1, 11).mapToObj(i -> "field" + i).collect(joining(","));
+            String valQ = IntStream.range(1, 11).mapToObj(i -> "'" + FIELD_VAL + "'").collect(joining(","));
+
+            String queryStr = String.format("insert into %s(%s, %s) values(?, %s);", TABLE_NAME, "ycsb_key", fieldsQ, valQ);
```
[the remark this answers is not in the document]
Comment (korlov42) [reply modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/InsertBenchmark.java]: fixed

**Review by xtern — APPROVED** [review]

File: modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/InsertBenchmark.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (korlov42) [reply modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/InsertBenchmark.java]: make sense. I've extracted this to the static method. Please take a look one more time

[the remark this answers is not in the document]
Comment (xtern) [reply modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/InsertBenchmark.java]: yes, that's what I meant, thanks

File: modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/SelectBenchmark.java [hunk 2]
```diff
@@ -195,4 +238,35 @@ public void tearDown() throws Exception {
...
+            IgniteUtils.closeAll(client);
+        }
+
+        KeyValueView<Tuple, Tuple> kvView() {
+            return kvView;
+        }
+    }
 }
+
+
```
Comment (AMashenkov) [inline modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/SelectBenchmark.java:272]:
```suggestion

```

**Pull request merged**